src/components/SetupFundersPage/SetupFundersPageTest.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
```

Comment:
This file needs to be renamed to SetupFundersPage.test.tsx, as it's not being picked up by Jest. However I'm not convinced it'll work.